benchmarks/transactions-generator/justfile

```
@@ -39,17 +44,15 @@ unlimit:
     jq '.gas_limit=20000000000000000' {{near_genesis_file}} > tmp_genesis.json && mv tmp_genesis.json {{near_genesis_file}}
     jq '.view_client_threads=8 \
      | .store.load_mem_tries_for_tracked_shards=true \
-     | .produce_chunk_add_transactions_time_limit={"secs": 0, "nanos": 800000000} \
+     | .produce_chunk_add_transactions_time_limit={"secs": 0, "nanos": 500000000} \
      | .transaction_request_handler_threads = 4' {{near_config_file}} > tmp_config.json \
     && mv tmp_config.json {{near_config_file}}
 
-do-it tps:
+do-it tps="20000" timeout="2m":
+    #!/bin/bash
```

```suggestion
    #!/usr/bin/env bash
```

`/bin/bash` is not an universal location for `bash` to be at. Even better if you use `require` to look-up `bash` from `$PATH`. Unless `require` doesn't work for some reason?